redis/src/main/scala/zio/redis/options/Shared.scala

```diff
@@ -18,6 +19,30 @@ trait Shared {
     case object SetGreaterThan extends Update
   }
 
+  sealed trait Expire { self =>
+    private[redis] final def stringify: String =
+      self match {
+        case Expire.SetExpireSeconds      => "EX"
+        case Expire.SetExpireMilliseconds => "PX"
+      }
+  }
+  object Expire {
+    case object SetExpireSeconds      extends Expire
+    case object SetExpireMilliseconds extends Expire
+  }
+
+  sealed trait ExpiredAt { self =>
+    private[redis] final def stringify: String =
+      self match {
+        case ExpiredAt.SetExpireAtSeconds      => "EXAT"
+        case ExpiredAt.SetExpireAtMilliseconds => "PXAT"
+      }
+  }
+  object ExpiredAt {
+    case object SetExpireAtSeconds      extends ExpiredAt
+    case object SetExpireAtMilliseconds extends ExpiredAt
+  }
+
```

Comment:
Is this really `shared` or should it be part of the `strings` options?

Reply:
Thanks, I'm not sure, just intuition.

Reply:
I suggest moving this to `string` options if these are only used there, maybe you can double-check. Otherwise, let's keep it like it is.